.golangci.yml

```diff
@@ -228,6 +244,38 @@ issues:
         - staticcheck
       text: "SA9003:"
 
+    - linters:
+        - golint
+      text: "package comment should be of the form"
+
+    - linters:
+        - golint
+      text: "don't use ALL_CAPS in Go names;"
+
+    - linters:
+        - golint
+      text: "don't use underscores in Go names;"
+
+    - linters:
+        - golint
+      text: "which can be annoying to use"
+
+    - linters:
+        - golint
+      text: "and that stutters; consider calling this"
+
+    - linters:
+        - golint
+      text: "should have comment or be unexported"
+
+    - linters:
+        - golint
+      text: "or a comment on this block"
+
+    - linters:
+        - golint
+      text: "error strings should not be capitalized or end with punctuation or a newline"
+
```

Comment:
Can you explain the `text` fields here? Some of them are unclear and confusing, for example, "or a comment on this block" or "which can be annoying to use". I am not see this in other projects or examples.

Reply:
Yes, that is what I explained yesterday in the call. This section is the linter special section to skip tests. Do something, comment out this section and run the linters, you will see what is it for right away on the linter error messages reported.